package.json

```diff
@@ -174,4 +174,4 @@
     "uglifyjs-webpack-plugin": "1.1.5"
   },
   "schematics": "./schematics/dist/collection.json"
-}
\ No newline at end of file
+}
```

Comment:
Please, revert whitespace/newline changes here and also in `package-lock.json` and `select-showcase.component.html`.